tests/test_ingesters.py

```diff
@@ -26,6 +26,8 @@ class IngesterTestCase(django.test.TransactionTestCase):
     """Test the base ingester class"""
 
     def setUp(self):
+        Parameter.objects.count = mock.Mock()
+        Parameter.objects.count.return_value = 2
```

Comment:
Please use patch for this, in order to limit unintended side effects

Reply:
Done in the next commit

Reply:
Done in the next commit